xml/docu_styleguide.structure.xml

```diff
@@ -302,6 +302,55 @@ default 0 <co xml:id="co-default"/></screen>
     </tbody>
    </tgroup>
   </table>
+
+  <para>You can group different callouts to point to the same annotation
+  so that it is not repeated. To do this, complete the following steps:</para> 
```

Comment:
I would separate the grouping and the explanation.

```suggestion
  <para>You can group different callouts to point to the same annotation.
   This helps to avoid repeating the same annotation.
   To do this, complete the following steps:</para> 
```